### ISSUE TITLE
Styling element outlines and borders in a way that removes or renders non-visible the visual focus indicator

### ISSUE THREAD

**Issue by will0684** [DTS-STN/Alpha-Site#179]
Focus outline on the "Skip to main content" link is not visible.

To test this, load the page and navigate to the skip link using the Tab key on your keyboard.

Note: it is recommended to remove the aria (role="button"), the skip menu items should be coded as links. 
Assistive technology users can navigate the pages using links, and for this theme, the skip menu is always coded as links.

Visit the (GCWeb, the WET-BOEW Canada.ca theme) for working example: https://wet-boew.github.io/GCWeb/index-en.html
